src/canvg.js

```diff
@@ -2321,6 +2324,9 @@ function build(opts) {
 
     this.adjustChildCoordinates = function(ctx, textParent, parent, i) {
       var child = parent.children[i];
+      if (!(child instanceof svg.Element.TextElementBase)) {
+        return child;
```

Comment:
prevents `<animate />` from running below code

Reply:
Could you perhaps duck-type here instead? Unless there is an ES6 `Symbol.hasInstance` (which does the duck-typing or what not), `instanceof` is not cross-frame safe...

Reply:
by `duck-type` you want `if (typeof(child.measureText) !== 'function')` or something similar?

Reply:
Yeah, exactly... If that method (or whatever combo of method/property checks can pin it down) is unique to that type...

Reply:
done